### PR TITLE
Handle error when ctlog unable to generate public key

### PR DESCRIPTION
**Pull request by osmman**
Solves [SECURESIGN-675](https://issues.redhat.com//browse/SECURESIGN-675)

### CODE REVIEW

File: controllers/ctlog/actions/handle_keys.go [hunk 1]
```diff
@@ -100,6 +100,9 @@ func (g handleKeys) Handle(ctx context.Context, instance *v1alpha1.CTlog) *actio
 			}
 		}
 		config, err = utils.GeneratePublicKey(&utils.PrivateKeyConfig{PrivateKey: private, PrivateKeyPass: password})
+		if err != nil || config == nil{
+			g.Failed(fmt.Errorf("unable to generate public key: %w", err))
```
Comment (bouskaJ) [inline controllers/ctlog/actions/handle_keys.go:104]:
```suggestion
			return g.Failed(fmt.Errorf("unable to generate public key: %w", err))
```

Comment (osmman) [reply controllers/ctlog/actions/handle_keys.go]: fixed

**Review by bouskaJ — APPROVED** [review]

**Pull request merged**